src/main.ts

```diff
@@ -0,0 +1,78 @@
+import path from 'path'
+import * as core from '@actions/core'
+import * as exec from '@actions/exec'
+import * as tc from '@actions/tool-cache'
+import * as utils from './utils'
+
+function getArgs(flag: string, input: string): string[] {
+  const value = core.getInput(input)
+  if (value) return [flag, value]
+  return []
+}
+
+async function run(): Promise<void> {
+  const yorVersion = core.getInput('version')
+  const commitChanges = core.getInput('commit_changes') === 'YES'
```

Comment:
Usually for boolean we just expect a more natural boolean, like `true`.
Github has `core.getBooleanInput`

Reply:
Agreed, let's change it